### ISSUE TITLE
script problem

### ISSUE THREAD

**Issue by doaaelshaarawy** [dicengine/dice#243]
![error-dice](https://user-images.githubusercontent.com/40676088/149195946-f5e500c2-682d-4cfc-82ca-73914d68729f.png)
this is a problem  face me when i enter more than 10000 frame in the deformed shape

**Comment by dicengine** [comment]
I will have to check what is happening with the user interface in this case, but in the mean time you can manually update the input.xml file to include all 10,000 frames and run DICe from the command prompt
`dice.exe -i input.xml`

**Comment by doaaelshaarawy** [comment]
thanks for reply
okay i will try this
(dice.exe -i input.xml) this in the code script not the  GUI ?

**Comment by dicengine** [comment]
Yes, that is the syntax to run DICe from the command line. You will probably have to supply the full path to where you installed DICe

`<path to DICe bin folder>\dice.exe -i input.xml`
